script.js

```diff
@@ -15,3 +15,117 @@ for (let i = 0; i < 3; i += 1) {
     document.body.classList.toggle('no-scroll');
   });
 }
+
+const projects = [
+  {
+    id: 1,
+    name: 'Multi-Post Stories',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    imageUrl: 'gymfit.png',
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    demoLink: '#',
```

Comment:
Please ensure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins).

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/162162200-1995d4f7-544f-4f4c-917b-93b7f386a33c.png) | ![image](https://user-images.githubusercontent.com/10332499/162161931-8cb80927-a2a4-4b41-8352-2bf72a09e473.png)

##Mobile version:
Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/162162591-999ccc49-ce07-4849-a0d5-d59bdb3ff429.png) | ![image](https://user-images.githubusercontent.com/10332499/162162510-028e5888-f103-46df-a4df-867840fa4b80.png)